lib/pf/scan/nessus6.pm

```diff
@@ -0,0 +1,196 @@
+package pf::scan::nessus6;
+
+=head1 NAME
+
+pf::scan::nessus6
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::scan::nessus6 is a module to add Nessus v6 scanning option.
+
+=cut
+
+use strict;
+use warnings;
+
+use Log::Log4perl;
+use Readonly;
+
+use base ('pf::scan');
+
+use pf::config;
+use pf::scan;
+use pf::util;
+use pf::node;
+use pf::constants::scan qw($SCAN_VID $PRE_SCAN_VID $POST_SCAN_VID $STATUS_STARTED);
+use Net::Nessus::REST;
+
+sub description { 'Nessus6 Scanner' }
+
+=head1 SUBROUTINES
+
+=over   
+
+=item new
+
+Create a new Nessus6 scanning object with the required attributes
+
+=cut
+
+sub new {
```

Comment:
I conquer but I think that would require refactor of the whole "scan" lib... For this specific PR, we can stick with that code and moosify it later